src/midi.js

```diff
@@ -5,6 +5,68 @@ const assert = require('assert');
 const jsmidgen = require('jsmidgen');
 const setMiddleC = require('./setMiddleC');
 
+function doesArrayContainAnArray(arr){
+  return arr.some(function(ele){
+    return Object.prototype.toString.call( ele ) === '[object Array]'
+  });
+}
+
+/**
+ * Takes an array of clips and merges them on the same track. 
+ * @param  {Array} clipArray	Format: [{note: ['c3'], level: 127, length: 64}, [...], ...]
+ */
+const mergeMidiClips = (clipArray) => {
+
+	let clip = [];
+	let maxLenght = 0;
+	// Get the length of the final clip to be returned. (longest clip)
+	clipArray.forEach((noteObj) => {
+		if (noteObj.length > maxLenght) {
+			maxLenght = noteObj.length;
+		} 
+	});
+	// loop through all clicks and get the value of each clip.
```

Comment:
No, I meant click. As in each time the for loops it's one note position.  Like if the bar is in 8ths then each loop represents what is played on one 8th click. Does that make sense?